lib/server.go

```diff
@@ -165,7 +165,7 @@ type Server struct {
 	// It can be used to find computational bottlenecks.
 	timer *Timer
 
-	stateChangeSyncer *StateChangeSyncer
+	StateChangeSyncer *StateChangeSyncer
```

Comment:
Why does this need to be public? just curious.

Reply:
I believe we access it in some of the testing routines we run in the state consumer testing suite.